.vsts-ci.yml

```diff
@@ -325,8 +323,10 @@ extends:
             buildArchitecture: arm64
             runTests: false
 
-        # Source Build
-        - template: /eng/common/templates-official/jobs/source-build.yml@self
```

Comment:
Intentional indentation change, though practically it makes no difference because the conditional it was incorrectly under would always be true for .vsts-ci.yml